### ISSUE TITLE
Watched History for TV Shows

### ISSUE THREAD

**Issue by ShaVerZ** [123Venom/plugin.video.venom#50]
Anyone having issues with <My TV Shows> <History> because mine has stopped logging anything watched after 16 April 2020?

<My Movies> <History> are fine and all up-to-date with no problem.

I checked Seren to see if the problem exists within that addon but all is fine and up-to-date.

I have re-authorised and re-synced Trakt but this does nothing to correct the issue.

Thanks in advance.....

**Comment by 123Venom** [comment]
I just checked mine with the two shows I watched last night and they're both there in my history.  What I did note however is to check your sorting option used under `TV Shows`.  It seems to follow the `Calendar` sort option and I don't think I personally like that at all.  Calendar sorting we want by `Premiered date` and that simply will not work with History and could by why it looks like it's missing current shows you've watched..they should be there just prob not at the top where expected.  I think the sort method should follow "Progress Sorting` option, or I make a new category but Progress and History to me seems we'd want both sorted by `Date Watched`
